core/pkg/resultshandling/printer/v2/resourcetable.go

```diff
@@ -112,17 +111,14 @@ func generateResourceRows(controls []resourcesresults.ResourceAssociatedControl,
 }
 
 func generateResourceHeader(short bool) []string {
-	var headers []string
+	headers := make([]string, 0)
```

Comment:
you could also just declare with a var...

Reply:
Agreed, it is just that make is slightly more performant than var declaration.